.github/ISSUE_TEMPLATE/new_source.md

```diff
@@ -0,0 +1,36 @@
+---
+name: New content
+about: Create a report for a new interview or speech source
+title: 'NEW CONTENT:'
+labels: content
+assignees: ''
+
+---
+
+## Summary
+
+title:
+<!-- e.g. CNN Town Hall -->
+
+date:
+<!-- Event date, ISO format preferred: YYYY-MM-DD -->
+
+description:
+<!-- Optional description of the event -->
+
+## Topics
+
+<!-- This section can be repeated multiple times -->
+
+topic:
+<!-- Content topic reference -->
+
+start time:
+<!-- Starting timestamp for topic reference, ISO format preferred: hh:mm:ss -->
+
+end time:
```

Comment:
I actually think we might not need this, I don't think we've been keeping track of it so far

Reply:
I thought it might be a nice addition to possibly create clips in the future or something, but have no problem taking it out.

Reply:
sure, let's leave it in and mention that it's optional. could be useful to display durations of the videos